### ISSUE TITLE
Possibility of including the original reference in element-citation

### ISSUE THREAD

**Issue by fabiobatalha** [pkp/ots#135]
Thinking in a production pipeline based on OTS -> Texture -> Final Version.

Be the reference metadata retrieved by Crossref, parsed by Cermine or MeTypeset, we still need a good display version of the references for publishing matters.

In most of the cases the parsed references have glitches of metadata that must be fixed before publishing any content, increasing the publishing time.

Including the original textual reference in the metadata allows publishers to implement more flexible production steps separating publication issues from the production of quality metadata for data analysis, dissemination, including developing their own automated quality control for the given structured data.

There is a related ticket in texture to deal with untagged references: https://github.com/substance/texture/issues/412

The basic idea is to have both metadata (structured, unestructured in the same element-citation).

``` xml
<element-citation>
    ...
    <styled-content specific-use="display|print|..." style-type="vancouver|abnt|iso">
    Reference.....
    </styled-content>
</element-citation>
```

**Comment by axfelix** [comment]
I wouldn't mind supporting this depending on the outcome of the Texture meeting, but I'll leave it until then.

**Comment by fabiobatalha** [comment]
Yes it is the best approach, I only put it here to not forget about it in the context of OTS.
